### DISCUSSION TITLE
firebasecore Include of non-modular header inside framework module

### DISCUSSION THREAD

**Discussion by MTEV** [firebase/flutterfire#11227] [Q&A]
i have this error when try. run 

#import <FirebaseCore/FIRApp.h>

Include of non-modular header inside framework module 'FirebaseCoreExtension.FIRAppInternal': '/Users/dev.user/Desktop/Project/mybroject/ios/Pods/Headers/Public/FirebaseCore/FIRApp.h'

and Could not build Objective-C module 'FirebaseCoreExtension'

my pod file 

platform :ios, '13.0'
source 'https://github.com/CocoaPods/Specs.git'

pod 'Firebase/Analytics'
pod 'Firebase/Auth', :modular_headers => true
pod 'Firebase/Firestore'
pod 'GTMAppAuth', :modular_headers => true
pod 'Firebase'
pod 'FirebaseCoreInternal', :modular_header => true
pod 'GoogleUtilities', :modular_headers => true
pod 'GoogleSignIn'
pod 'FirebaseCore', :modular_header => true
pod 'Firebase/Messaging', :modular_header => true

any help please

**Comment by Finni123** [comment]
I have the same problem, did you find a solution?

EDIT: I am hoping to find a solution [in this issue](https://github.com/invertase/firestore-ios-sdk-frameworks/issues/100)

**Reply by habyaad** [reply]
how did you fix it?
